aws-ts-eks/README.md

```diff
@@ -12,6 +12,7 @@ To deploy your infrastructure, follow the below steps.
 1. [Install Pulumi](https://pulumi.io/install)
 2. [Install Node.js 8.11.3](https://nodejs.org/en/download/)
 3. [Configure AWS Credentials](https://pulumi.io/install/aws.html)
+4. [Install `aws-iam-authenticator`](https://docs.aws.amazon.com/eks/latest/userguide/getting-started.html)
```

Comment:
Unfortunate we have to link to this giant page with all the complexity of installing EKS.  Any way we can deep link or link to some other download page for this tool?

Reply:
Believe it or not, that is the _least_ intimidating set of installation instructions for `aws-iam-authenticator`. There is a slightly better deep link, though: https://docs.aws.amazon.com/eks/latest/userguide/getting-started.html#get-started-kubectl